src/net.cpp

```diff
@@ -797,7 +812,7 @@ CNetMessage V1TransportDeserializer::GetMessage(const std::chrono::microseconds
     return msg;
 }
 
-void V1TransportSerializer::prepareForTransport(CSerializedNetMsg& msg, std::vector<unsigned char>& header) const
+bool V1TransportSerializer::prepareForTransport(CSerializedNetMsg& msg, std::vector<unsigned char>& header) const
```

Comment:
1f22117: it'd be easier to understand if we renamed `header` to `transport_header`.

Reply:
Going to keep this as-is for now as it'd be less confusing for v2 and perhaps more confusing for v1.